src/pages/MovieDetail/Components/MovieDetailContentsSection/MovieDetailContentsSection.js

```diff
@@ -1,8 +1,46 @@
 import React from 'react';
+import Comment from './Components/Comment/Comment';
+
 import './MovieDetailContentsSection.scss';
 
 export default class MovieDetailContentsSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.commentList = React.createRef();
+    this.count = React.createRef(0);
+    this.count.current = 0;
+    this.state = { commentListContents: [] };
+  }
+
+  goToPrevious = () => {
+    const { style } = this.commentList.current;
+    if (this.count.current === 0) return;
+    this.count.current -= 1;
```

Comment:
굳이 -= 연산자 활용할 필요는 없는 케이스같아서 
```suggestion
    this.count.current--;
```
가 좀 더 직관적으로 보이네요!

Reply:
넵 감사합니다!!